pkg/crc/version/version.go

```diff
@@ -67,8 +68,11 @@ func GetCommitSha() string {
 	return commitSha
 }
 
-func GetBundleVersion(_ crcPreset.Preset) string {
-	return bundleVersion
+func GetBundleVersion(preset crcPreset.Preset) string {
+	if preset == crcPreset.OpenShift {
+		return bundleVersion
+	}
+	return okdVersion
 }
```

Comment:
Not fully clear why we have a GetBundleVersion depending on preset, but only for OpenShift/OKD. For Podman, one has to use GetPodmanVersion instead :-/

Reply:
@cfergeau I will take care of that also, ideally `GetBundleVersion` should return any preset bundle version (okd/ocp/podman or microshift ..etc.). Not with this PR as of now.

Reply:
I'd just go with `GetOKDVersion()` for now, this way there's no way to misuse/misunderstand `GetBundleVersion(preset)`

Reply:
adding a `GetOKDVersion()` helper and not importing the presets on the version package will be good, the user of the version package can then decide which bundle version (okd/ocp/podman) to use depending on the preset